wp-content/themes/core/components/section_nav/Section_Nav_Controller.php

```diff
@@ -0,0 +1,220 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Project\Templates\Components\section_nav;
+
+use Tribe\Libs\Utils\Markup_Utils;
+use Tribe\Project\Templates\Components\Abstract_Controller;
+use Tribe\Project\Templates\Components\button\Button_Controller;
+use Tribe\Project\Templates\Components\Deferred_Component;
+use Tribe\Project\Templates\Components\navigation\Navigation_Controller;
+use Tribe\Project\Templates\Components\text\Text_Controller;
+
+class Section_Nav_Controller extends Abstract_Controller {
+
+	public const ATTRS             = 'attrs';
+	public const CLASSES           = 'classes';
+	public const CONTAINER_ATTRS   = 'container_attrs';
+	public const CONTAINER_CLASSES = 'container_classes';
+	public const MENU              = 'menu';
+	public const MOBILE_LABEL      = 'mobile_label';
+	public const DESKTOP_LABEL     = 'desktop_label';
+	public const MORE_LABEL        = 'more_label';
+	public const STICKY            = 'sticky';
+	public const MOBILE_INIT_OPEN  = 'mobile_init_open';
+
+	/**
+	 * @var string[]
+	 */
+	private array $attrs;
+
+	/**
+	 * @var string[]
+	 */
+	private array $classes;
+
+	/**
+	 * @var string[]
+	 */
+	private array $container_attrs;
+
+	/**
+	 * @var string[]
+	 */
+	private array $container_classes;
+
+	/**
+	 * @var int|string|\WP_Term
+	 */
+	private $menu;
+
+	/**
+	 * @var string
```

Comment:
🟠  Should fix: If it's a single value like this, the type is all that is required.